force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +91,18 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+        List<String> sentenceWords = sentence.split(' ');
 
-        // Create a set to store the unique words
-
-        // Add each word to the set
+        // Create a set to store the unique words and add each word to the set
+        Set<String> uniqueWordsSet = new Set<String>();
+        for (String uniqueWord : sentenceWords){
+            uniqueWordsSet.add(uniqueWord);
+        }
```

Comment:
Set has a constructor which takes a List and builds a Set, removing duplicate elements.